build/version.gni

```diff
@@ -1,27 +1,11 @@
-# Copyright 2015 The Chromium Authors. All rights reserved.
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-# This exposes the Chrome version and Crosswalk version as GN variables for
```

Comment:
I think it's valid to keep the first half of this file as-is, and only extend it at the end with the Android bits.